inputs/prod_inner.json

```diff
@@ -71,9 +71,6 @@
       "name": "inject_yum_repo"
     },
     {
-      "name": "dockerfile_content"
```

Comment:
I wonder if we should still keep this here but add 'required':false.
Then it will work with buildroots that don't supply that plugin, but also with buildroots that require it.